platform/android/tests/docs/EXERCISER_TESTS.md

```diff
@@ -2,22 +2,19 @@
 
 UI/application exerciser tests are stress test using random generated users events.
 
-##Running Locally
+# Monkey Exerciser
 
-The Android SDK provides a test tool called [Monkey](http://developer.android.com/tools/help/monkey.html),
+The Android SDK provides a test tool called [UI/Application Monkey Exerciser](https://developer.android.com/studio/test/monkey),
 "a program that runs on your emulator or device and generates pseudo-random streams of user events
 such as clicks, touches, or gestures, as well as a number of system-level events."
 
-To exercise Monkey on the test app, install the package on the device (e.g. via Android Studio)
-and then:
+## Setup
 
-```
-$ adb shell monkey -p com.mapbox.mapboxgl.testapp -v 500
-```
-
-##Running on AWS Device Farm
-
-Amazon Device farm supports a similar tool called `Built-in Fuzz Test`.
-"The built-in fuzz test randomly sends user interface events to devices and then reports results."
-
-More information about [Built-in Fuzz Test](http://docs.aws.amazon.com/devicefarm/latest/developerguide/test-types-built-in-fuzz.html)
\ No newline at end of file
+ - install the qa debug build variant of the test application
+ - pin app on screen (this avoid opening statusbar through monkey exerciser)
+   - activate with `settings>security>screen pinning`
+   - press the recents apps button
+   - each app will support a configuration to pin it 
+ - run an monkey exerciser with a commands as:
+   - `adb shell monkey -p com.mapbox.mapboxsdk.testapp.monkey -v --pct-nav 10 --pct-syskeys 2 --pct-anyevent 2 --pct-touch 44 --pct-motion 40 --pct-trackball 2 --monitor-native-crashes 20000`
```

Comment:
Follow up request - could we include the adb command in the gradle script to be executed after the app is installed? Something like [finalizer](https://docs.gradle.org/current/userguide/more_about_tasks.html#sec:finalizer_tasks) might do the trick. This is not a priority though.

Reply:
like introducing a gradle task specific for running monkey tests?

Reply:
For example `installQa.finalizedBy runMonkey` where `runMonkey` is and `exec` block. This would start the monkey test straight after installing the apk, like an instrumentation test